templates/support/index.html

```diff
@@ -1079,14 +1078,29 @@ <h2>Support for AWS and Azure with Ubuntu Pro</h2>
               height="75",
               hi_def=True,
               loading="lazy",
-              attrs={"class": "u-align--center", "id": ""},
           ) | safe
         }}
       </div>
       <p>
         <a href="/azure">Learn more about Ubuntu Pro on Azure&nbsp;&rsaquo;</a>
       </p>
     </div>
+    <div class="col-4">
+      <div class="u-sv2 u-align--center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/6e176d9a-Google+Cloud+stacked.svg",
+          alt="Google cloud",
```

Comment:
This logo doesn't look good. I requested them an asset link, let's see what they say in the copy docs.

Reply:
updated the logo url but it looks a bit small. Please let me know if the size needs to be changed!

Reply:
Did you try increasing the `width` and `height`?

Reply:
Looks like the height needs to remain at 75 like the rest of the logos. You may need to adjust the width so it look proportionate. Or you can take the original size of the image and calculate a ratio `original width / original height` and then multiple by 75

Reply:
updated the `width` :)

Reply:
mm it still looks way too small I think... Maybe a designer can help us @anasereijo or @spencerbygraves , can you advise? should we change to a different logo or allow the height to push down the text?

Reply:
@minkyngkm try this? https://manager.assets.ubuntu.com/?q=6e176d9a

Reply:
Updated the img src with the one @pmahnke  provided. Does it look better @carkod ? Also, updated the height of all the 3 logo images to 75 and resized each image's original width proportionally to the height.